ci/loadWin.groovy

```diff
@@ -55,9 +57,42 @@ def cleanup_directories() {
     }
 }
 
+def deleteOldDirectories() {
+    command = 'forfiles /P c:\\ /D -14 | grep -oE "(PR-[0-9]*)"'
+    status = bat(returnStatus: true, script: command)
+    if ( status != 0) {
+        println "No PR-XXXX older than 14 days for cleanup."
+        return
+    }
+
+    def existing_prs_string = bat(returnStatus: false, returnStdout: true, script: command)
+
+    println existing_prs_string
+
+    def existing_prs = existing_prs_string.split(/\n/)
+
+    for (int i = 0; i < existing_prs.size(); i++) {
+        // Check for empty output, Part of output contains the command that was run
+        if ( existing_prs[i] == null || existing_prs[i].allWhitespace || existing_prs[i].toLowerCase().contains("grep")) { continue }
+        // Check if directory was created more than 14 days ago
```

Comment:
Why is this comment here? Doesn't `existing_prs` contain only 14 days long dirs?